CHANGELOG.md

```diff
@@ -9,11 +9,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - [2135](https://github.com/FuelLabs/fuel-core/pull/2135): Added metrics logging for number of blocks served over the p2p req/res protocol.
 - [2151](https://github.com/FuelLabs/fuel-core/pull/2151): Added limitations on gas used during dry_run in API.
+- [2188](https://github.com/FuelLabs/fuel-core/pull/2188): Added the new variant `V2` for the `ConsensusParameters` which contains the new `block_transaction_size_limit` parameter.
 - [2163](https://github.com/FuelLabs/fuel-core/pull/2163): Added runnable task for fetching block committer data.
 
 ### Changed
 
 #### Breaking
+- [2199](https://github.com/FuelLabs/fuel-core/pull/2199): Applying several breaking changes to the WASM interface from backlog:
```

Comment:
This description shouldn't be in this PR.

Reply:
True, it's not necessarily a part of the feature being implemented in this PR, but the breaking changes was pulled into this branch [here](https://github.com/FuelLabs/fuel-core/pull/2188/commits/b7d8da650d9c8effe55d76b4d95b742ffb11da09) and [here](https://github.com/FuelLabs/fuel-core/pull/2188/commits/dba42fc76b496ddc02b72492cdbe3b61c8e966c6), so technically, this message belongs here.

I'll most likely not merge this PR before [the one which contained the original change](https://github.com/FuelLabs/fuel-core/pull/2199) is merged first.